tests/e2e/cucumber/features/journeys/kindergarten.feature

```diff
@@ -41,26 +41,26 @@ Feature: Kindergarten can use web to organize a day
-      | groups/Pre-Schools Pirates/meal plan               | Carol     | user  | Can edit                              | folder       |
-      | groups/Pre-Schools Pirates/meal plan/lorem-big.txt | sales     | group | Can view                              | file         |
-      | groups/Pre-Schools Pirates/meal plan/lorem-big.txt | Carol     | user  | Can view                              | file         |
-      | groups/Kindergarten Koalas/meal plan               | sales     | group | custom_permissions:read               | folder       |
-      | groups/Kindergarten Koalas/meal plan               | security  | group | custom_permissions:read,update,delete | folder       |
-      | groups/Kindergarten Koalas/meal plan/lorem.txt     | sales     | group | custom_permissions:read               | file         |
-      | groups/Kindergarten Koalas/meal plan/lorem.txt     | security  | group | custom_permissions:read               | file         |
-      | groups/Teddy Bear Daycare/meal plan                | Brian     | user  | custom_permissions:read,update,create | folder       |
-      | groups/Teddy Bear Daycare/meal plan                | Carol     | user  | custom_permissions:read,update,create | folder       |
-      | groups/Teddy Bear Daycare/meal plan/data.zip       | Brian     | user  | custom_permissions:read,update        | file         |
-      | groups/Teddy Bear Daycare/meal plan/data.zip       | Carol     | user  | custom_permissions:read,update        | file         |
+      | resource                                           | recipient | type  | role     | resourceType |
+      | groups/Pre-Schools Pirates/meal plan               | Brian     | user  | Can edit | folder       |
+      | groups/Pre-Schools Pirates/meal plan               | Carol     | user  | Can edit | folder       |
+      | groups/Pre-Schools Pirates/meal plan/lorem-big.txt | sales     | group | Can view | file         |
+      | groups/Pre-Schools Pirates/meal plan/lorem-big.txt | Carol     | user  | Can view | file         |
+      | groups/Kindergarten Koalas/meal plan               | sales     | group | Can view | folder       |
+      | groups/Kindergarten Koalas/meal plan               | security  | group | Can edit | folder       |
+      | groups/Kindergarten Koalas/meal plan/lorem.txt     | sales     | group | Can view | file         |
+      | groups/Kindergarten Koalas/meal plan/lorem.txt     | security  | group | Can view | file         |
+      | groups/Teddy Bear Daycare/meal plan                | Brian     | user  | Can edit | folder       |
+      | groups/Teddy Bear Daycare/meal plan                | Carol     | user  | Can edit | folder       |
+      | groups/Teddy Bear Daycare/meal plan/data.zip       | Brian     | user  | Can edit | file         |
+      | groups/Teddy Bear Daycare/meal plan/data.zip       | Carol     | user  | Can edit | file         |
```

Comment:
here we wanted to check sharing and share updating with custom roles

in the sharing ng we can share using roles and permissions action:

 ```
"roles": [
    "1c996275-f1c9-4e71-abdf-a42f6495e960"
  ]
  "@libre.graph.permissions.actions": [
    "libre.graph/driveItem/permissions/create",
    "libre.graph/driveItem/quota/read"
  ]
```

If we leave the `permissionsAction` feature (sharing with custom permissions) on the web, I wouldn't change this test. if you need a help with e2e just ping me

or we can skip this tests temporaly and change them in separate PR/PRs

Reply:
As discussed with @tbsbdr , we won't have custom permissions with sharing NG anymore. That's why I replaced the custom permissions in this test with predefined roles.